packages/logger/src/Logger.ts

```diff
@@ -766,6 +782,9 @@ class Logger extends Utility implements ClassThatLogs {
     } = options;
 
     this.setEnvVarsService();
+    // order is important, it uses EngVarsService()
+    this.initConsole();
```

Comment:
Should we call this `setConsole` like the other methods? Mostly for consistency but also because we are not always initializing. Thoughts?

Reply:
I agree, naming is one of two hard things in CS 😀